app/models/config.rb

```diff
@@ -10,15 +10,16 @@ class Config
   CONFIG_FIELDS = [
     :insurance, :external_pledge_source, :pledge_limit_help_text,
     :language, :resources_url, :fax_service, :referred_by,
-    :practical_support
+    :practical_support, :start_of_week
   ].freeze
 
   # Define overrides for particular config fields.
   # Useful if there is no `_options` method.
   HELP_TEXT_OVERRIDES = {
     resources_url: 'A link to a Google Drive folder with CM resources. ' \
                    'Ex: https://drive.google.com/drive/my-resource-dir',
-    fax_service: 'A link to your fax service. ex: https://www.efax.com'
+    fax_service: 'A link to your fax service. ex: https://www.efax.com',
+    start_of_week: 'Ex: Monday'
```

Comment:
Looking at the UI I think we should make this a little bit clearer:

![image](https://user-images.githubusercontent.com/3866868/59568695-56d19200-904c-11e9-8d71-21e53c2d5cd0.png)

I think this might be a little clearer to someone looking for clarity on what this changes:

```suggestion
    start_of_week: 'The start day of your budget week, if it doesn't start on Monday. ex: Sunday.
```

Please smash if it looks good to you!